packages/bpk-component-barchart/src/__snapshots__/BpkChartAxis-test.js.snap

```diff
@@ -435,7 +435,7 @@ exports[`BpkChartAxis X should render with label 1`] = `
         class="bpk-chart__axis-label"
         text-anchor="middle"
         x="60"
-        y="40"
+        y="34"
```

Comment:
Why have these change? 🤔

Reply:
Let me check into this as you are right it shouldn't have and might make sense with some of the changes in this PR now

Reply:
Turns out these should be the correct values and what happened is it was actually broken 😆